tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/dockerized/model/Type.java

```diff
@@ -22,7 +22,15 @@
  */
 public enum  Type {
```

Comment:
I think it might be useful distinguishing between zipkin and apm - we might want to add additional verification capabilities?